### ISSUE TITLE
Update isolate_example for updated release

### ISSUE THREAD

**Issue by domesticmouse** [flutter/samples#653]
From [failed CI run](https://github.com/flutter/samples/pull/651/checks?check_run_id=1718863765#step:5:390):

```
== Testing 'isolate_example' on Flutter's  channel ==
~/work/samples/samples/isolate_example ~/work/samples/samples
Running "flutter pub get" in isolate_example...                    842ms
Analyzing isolate_example...                                    

   info • 'showSnackBar' is deprecated and shouldn't be used. Use ScaffoldMessenger.showSnackBar. This feature was deprecated after v1.23.0-14.0.pre. • lib/performance_page.dart:91:30 • deprecated_member_use
   info • 'showSnackBar' is deprecated and shouldn't be used. Use ScaffoldMessenger.showSnackBar. This feature was deprecated after v1.23.0-14.0.pre. • lib/performance_page.dart:105:30 • deprecated_member_use
```

**Comment by Abhishek01039** [comment]
@domesticmouse 
I would like to work on this issue.

**Comment by RedBrogdon** [comment]
I've brought the beta branch up to date so this can be landed there.

**Comment by RedBrogdon** [comment]
Closed by #679.
